fundamentals/arrays/lesson_notes/lesson_02.md

```diff
@@ -0,0 +1,108 @@
+# Reference Types
```

Comment:
I am concerned that passing by reference is too much information right now. It may be better to revisit this topic towards the end of the module.

Reply:
If we move it later, it becomes difficult to talk about array methods like `.slice()` and `.splice()` without ignoring parts of how they work. What benefit do students gain by waiting a day to talk about this?